jung-samples/src/main/java/edu/uci/ics/jung/samples/AddNodeDemo.java

```diff
@@ -82,20 +80,6 @@ public AddNodeDemo() {
     vv.getRenderContext().setVertexLabelTransformer(Object::toString);
     vv.setForeground(Color.white);
 
-    vv.addComponentListener(
```

Comment:
please modify the PR description to clarify that you're also cleaning up code like this that is no longer needed (and why it's no longer needed)

(no need to mention removal of commented-out code, but I'd like to be able to tell from the commit comment why this was removed, later)

Reply:
I put it back, but done the 'right' way, and added a comment saying why it's there